app/models/event.rb

```diff
@@ -147,6 +147,41 @@ def ready_to_schedule?(schedule_action)
     return true
   end
 
+  def self.staffing_level
+    begin
+      relevant_events = self.current_or_next_events
+      event_staffing = relevant_events.map { |event| event.staffing_level }
+      event_staffing.min
+    rescue => e
+      puts e
+      -1
+    end
+  end
+
+  def self.current_or_next_events
+    current_events = self.current_events
+    events = current_events.empty? ? [self.next_event] : current_events
+  end
+
+  def self.current_events
+    self.where("start_time < ?", Time.now)
```

Comment:
We'll need to add a check on status here.

Reply:
okay.  possible statuses are: ['Scheduled', 'In-session', 'Completed', 'Cancelled', "Closed"].  I assume that cancelled and closed events are the only ones that should not be included?